packages/cosmo/package.py

```diff
@@ -228,11 +220,11 @@ def install(self, spec, prefix):
     def test(self):
         if '~serialize' in self.spec:
             try:
-                subprocess.run([self.build_directory + '/test/tools/test_cosmo.py', str(self.spec), '.'], stderr=subprocess.STDOUT, check=True)
+                subprocess.run([self.build_directory + '/test/tools/test_cosmo.py', str(self.spec), '.'], stderr=subprocess.STDOUT, check=True, env=None)
             except:
                 raise InstallError('Testsuite failed')
         if '+serialize' in self.spec:
             try:
-                subprocess.run([self.build_directory + '/test/tools/serialize_cosmo.py', str(self.spec), '.'], stderr=subprocess.STDOUT, check=True)
+                subprocess.run([self.build_directory + '/test/tools/serialize_cosmo.py', str(self.spec), '.'], stderr=subprocess.STDOUT, check=True, env=None)
```

Comment:
This might be too drastic.

Reply:
I thought doing env=None was exactly doing what I wanted, that is giving the actual env to the subprocess, env=os.env strangely doesn't work, it's not very logic, please tell me if I am wrong.

Reply:
Even though `env=None` is the default value, it's apparently needed to work. We'll leave it as is for now.